src/Tribe/Tickets.php

```diff
@@ -4720,6 +4730,36 @@ public function deactivate(): void {
 			unset( self::$active_modules[ get_class( $this ) ] );
 		}
 
+		/**
+		 * Filter providers for the all tickets table.
+		 *
+		 * @since TBD
+		 *
+		 * @param string[] $provider_options The list of provider options.
+		 *
+		 * @return string[] The filtered list of provider options.
+		 */
+		public function filter_all_tickets_table_provider_options( $provider_options ) {
+			$provider_options[ $this->ticket_object ] = $this->plugin_name;
+
+			return $provider_options;
+		}
+
+		/**
+		 * Filter event meta keys for the all tickets table.
+		 *
+		 * @since TBD
+		 *
+		 * @param string[] $event_meta_keys The list of event meta keys.
+		 *
+		 * @return string[] The filtered list of event meta keys.
+		 */
+		public function filter_all_tickets_table_event_meta_keys( $event_meta_keys ) {
+			$event_meta_keys[ $this->ticket_object ] = $this->get_event_key();
+
+			return $event_meta_keys;
+		}
+
 		// @codingStandardsIgnoreEnd
```

Comment:
These functions are inside the `// @codingStandardsIgnore` for the deprecated methods - move them up and out of that please. (and any others that were put there inadvertently)